src/card.tsx

```diff
@@ -723,3 +729,151 @@ export const DisplayDataCard = React.forwardRef<HTMLDivElement, DisplayDataCardP
+                        >
+                            <Stack space={24}>
+                                <div>
+                                    <Stack space={8}>
+                                        {(headline || pretitle || title) && (
+                                            <header>
+                                                <Stack space={16}>
+                                                    {headline}
+                                                    <Stack space={4}>
+                                                        {pretitle && (
+                                                            <Text2
+                                                                forceMobileSizes
+                                                                truncate={pretitleLinesMax}
+                                                                as="div"
+                                                                regular
+                                                                textShadow={textShadow}
+                                                            >
+                                                                {pretitle}
+                                                            </Text2>
+                                                        )}
+                                                        <Text
+                                                            desktopSize={20}
+                                                            mobileSize={18}
+                                                            mobileLineHeight="24px"
+                                                            desktopLineHeight="28px"
+                                                            truncate={titleLinesMax}
+                                                            weight={textPresets.cardTitle.weight}
+                                                            as="h3"
+                                                        >
+                                                            {title}
+                                                        </Text>
+                                                    </Stack>
+                                                </Stack>
+                                            </header>
+                                        )}
+                                        {description && (
+                                            <Text2
+                                                forceMobileSizes
+                                                truncate={descriptionLinesMax}
+                                                as="p"
+                                                regular
+                                                textShadow={textShadow}
+                                            >
+                                                {description}
```

Comment:
Specs say that description use `textSecondaryInverse`
![image](https://user-images.githubusercontent.com/6722153/231277103-f808d54e-d22c-49d6-9f7d-32a99ccdd088.png)

BUT, this cause problems in dark mode because textPrimaryInverse is darker than textPrimaryInverse, this problem was opened by @kydorn in [this thread](https://teams.microsoft.com/l/message/19:29466ca25d9b4b5e82f023545edae231@thread.skype/1681210133382?tenantId=9744600e-3e04-492e-baa1-25ec245c6f10&groupId=bef9be07-03e4-4afb-8a79-f720a63e4c28&parentMessageId=1681210133382&teamName=Novum&channelName=DesignCore-Public&createdTime=1681210133382&allowXTenantAccess=false)

So.. maybe should we change this to textPrimaryInverse in displayMedia and poster? @aweell

Reply:
Agreed, textSecondaryInverse should be either textPrimaryInverse or inverse in poster and displayMedia

Reply:
choose one of the two 🙏😅. I'd change only the poster here, and I'd open a new ticket for displayMedia if you agree with that

Reply:
agree with @brtbrt. This PR is for PosterCard, if we want changes in the other cards, we can handle them in a different task

Reply:
yes is a different task. Just I was trying to align this problem

we will create a new ticket for that